src/main/java/calculator/FormulaGenerator.java

```diff
@@ -0,0 +1,42 @@
+package calculator;
+
+public class FormulaGenerator {
+
+    private String formula;
+
+    public FormulaGenerator(String input) {
+
+        this.formula = makeFormula(input);
+    }
+
+    private String makeFormula(String input) {
+        if (input.isEmpty()) return "0";
+        if (input.charAt(0) == '/') {
+            InputParser inputParser = new InputParser(input);
+            input = inputParser.getParsedInput();
+        }
+        for (int i = 1; i < input.length(); i++) {
+            input = changeSeparator(input, i);
+        }
+        return input;
+    }
+
+    private String changeSeparator(String input, int index) {
+        if (isSeparate(input.charAt(index))) {
+            input = input.replace(input.charAt(index), changeSeparatorToPlusSign());
+        }
+        return input;
+    }
+
+    public String getFormula() {
+        return formula;
+    }
+
+    private boolean isSeparate(char input) {
+        return !Character.isDigit(input);
+    }
+
+    private char changeSeparatorToPlusSign() {
+        return '+';
```

Comment:
멤버 변수에 상수로 선언하는 방법도 좋을 것 같습니다!
```suggestion
private static final char PLUS_SIGN = "+";
```